src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/TagHelperParseTreeRewriterTest/AllowsPrefixedTagHelpers7.stree.txt

```diff
@@ -1,6 +1,6 @@
 RazorDocument - [0..21)::21 - [<!th:myth></!th:myth>]
     MarkupBlock - [0..21)::21
-        MarkupElement - [0..10)::10
+        MarkupElement - [0..21)::21
```

Comment:
So I assume the change here is due a to a bug fix?

Reply:
Yes